arc-mlir/src/include/arc/arc-dialect.h

```diff
@@ -23,21 +23,94 @@
 #ifndef ARC_DIALECT_H_
 #define ARC_DIALECT_H_
 
+#include "mlir/IR/StandardTypes.h"
+#include <mlir/IR/Builders.h>
 #include <mlir/IR/Dialect.h>
 #include <mlir/IR/Function.h>
-#include <mlir/IR/Builders.h>
 #include <mlir/IR/Operation.h>
```

Comment:
@frej `clang-format` did this, should I revert it or does it matter?

Reply:
I have probably missed to run clang-format on that one, leave that as a separate patch as its unrelated to appenders